layouts/index.html

```diff
@@ -0,0 +1,40 @@
+{{ define "main" }}
+    {{ if .Site.Params.hero }}
+    {{ partial "hero.html" . }}
+    {{ end }}
+
+    {{ if .Site.Params.section1 }}
+    {{ partial "section1.html" . }}
```

Comment:
I think it is better to write better file name like instead of `section1.html` we can write `section_section_title_name.html`.

Reply:
`section1` is a hugo-fresh convention. It's only seen on the homepage, and it's the first section after the hero. We can rename it, but I think the naming convention you proposed is too much since this is only on the homepage. Ideas for a new name?